src/snowflake/connector/file_util.py

```diff
@@ -34,7 +34,7 @@ def get_digest_and_size(src: IO[bytes]) -> Tuple[str, int]:
             Tuple of src's digest and src's size in bytes.
         """
         use_openssl_only = os.getenv("SF_USE_OPENSSL_ONLY", "False") == "True"
-        CHUNK_SIZE = 16 * 4 * 1024
+        CHUNK_SIZE = 1024 * 1024
```

Comment:
Why did the chunk size change here? Can you add a comment. Also if this is the new chunk size can you replace this value with the megabyte constant?

Reply:
This was left over from testing even bigger block sizes, I'll change this back